src/external/jar_mod.h

```diff
@@ -1130,7 +1130,7 @@ static bool jar_mod_load( jar_mod_context_t * modctx, void * mod_data, int mod_d
     {
         if( modctx )
         {
-            memcopy(&(modctx->song.title),modmemory,1084);
```

Comment:
It might be a weird attempt to move the entire structure, and otherwise it looks like a memory fault waiting to happen.  The shortened size appears to be correct.

The use of a hand-built `memcopy` instead of the standard `memcpy` is also peculiar.  Optimizing compilers will take advantage of hardware copy instructions for `memcpy` in `string.h`.

Reply:
Mmmh... that's weird...